readme.md

```diff
@@ -2,6 +2,11 @@
 
 > Check TypeScript type definitions
 
+## Overview
```

Comment:
Can you move this below `## Install`? And leave 2 blank lines between the sections?

Reply:
@SamVerschueren done!